android/app/build.gradle

```diff
@@ -62,6 +64,7 @@ flutter {
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     testImplementation 'junit:junit:4.12'
-    androidTestImplementation 'androidx.test:runner:1.1.1'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
+    androidTestImplementation 'androidx.test:runner:1.2.0'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
+    implementation 'com.android.support:multidex:1.0.3'
```

Comment:
Since androidx is being used, modify the line to
`implementation "androidx.multidex:multidex:1.0.3" `

Reference [here](https://developer.android.com/studio/build/multidex).

Reply:
ok, will do